ubuntu_image/tests/test_parser.py

```diff
@@ -308,37 +309,7 @@ def test_size_offset_suffix(self):
         partition0 = volume0.structures[0]
         self.assertEqual(partition0.size, MiB(3))
 
-    def test_mbr_structure(self):
```

Comment:
We'll probably want to keep these tests of course, to verify legacy (deprecated) behavior.